src/components/project/project.service.ts

```diff
@@ -607,6 +604,7 @@ export class ProjectService {
       modifiedAt: DateTime.local(),
       ...input,
     };
+    const canEdit = createInput.status === ProjectStatus.InDevelopment;
```

Comment:
All projects start with their status as in development. Whether the fields can currently be edited is based on this changing `status`, so it probably needs to be done on query. We currently haven't integrated workflow into this permission model so @michaelmarshall might want to chime in here.